app/views/admin/editions/_search_results.html.erb

```diff
@@ -74,10 +77,14 @@
               <% end %>
             <% end %>
           </td>
-          <td class="author"><%= linked_author(edition.last_author) %></td>
-          <td class="updated"><span title="<%= edition.updated_at %>"><%= time_ago_in_words edition.updated_at %> ago</span></td>
-          <% if viewing_all_active_editions? %>
-            <td class="state"><%= admin_edition_state_text(edition) %></td>
+          <% if edition.locked? %>
+            <td colspan="3">Moved to Content Publisher</td>
```

Comment:
I'd suggest doing this with `text-align: center` (assuming that's cool with Sonia)

Reply:
Sonia doesn't mind either way so will change.